### PR TITLE
Hourly Rate breakdown

### PR DESCRIPTION
**Pull request by TobiA34**
- Added hourly rate breakdown
- Added hourly rate delegate
- Added total hourly view computed property 
- Added case hourly and link
- Removed national insurance link
- Created constant hourly and passed through case hourly  to calculate hourly salary

Before 
image

After
image

### CODE REVIEW

**Comment by tunds** [comment]
@TobiA34 lets add in some unit & ui tests for hourly lemme know once you've added it in.

**Review by tunds — APPROVED** [review]

**Comment by tunds** [comment]
Looks good to me, nice work

**Comment by TobiA34** [comment]
I will add the unit test for hourly then merge it into main.

**Comment by tunds** [comment]
> I will add the unit test for hourly then merge it into main.

Let me review it once you add in the tests 👌🏾

**Comment by TobiA34** [comment]
I have added the unit test for hourly and pushed it

**Comment by tunds** [comment]
Rather than creating an instance of double you should just be declaring the type i.e let salary: Double = 5400

Also you should look into setting up vals in the setup and tear down functions for unit tests

File: SalaryAppTests/SalaryAppTests.swift [hunk 1]
```diff
@@ -12,21 +12,27 @@ final class SalaryAppTests: XCTestCase {
 
     
     let salaryVm = SalaryViewModel()
```
Comment (tunds) [inline SalaryAppTests/SalaryAppTests.swift]: Follow what i said before and create this in the setup function & dispose of it in the tear down read up on it if you dunno what i mean

**Pull request merged**